tests/platforms/macOS/app/test_run.py

```diff
@@ -8,13 +8,19 @@
 from briefcase.platforms.macOS.app import macOSAppRunCommand
 
 
-def test_run_app(first_app_config, tmp_path):
+def test_run_app(first_app_config, tmp_path, monkeypatch):
     """A macOS app can be started."""
     command = macOSAppRunCommand(base_path=tmp_path)
     command.subprocess = mock.MagicMock()
     log_stream_process = mock.MagicMock()
     command.subprocess.Popen.return_value = log_stream_process
 
+    monkeypatch.setattr(
+        "briefcase.platforms.macOS.get_process_id_by_command", lambda *a, **kw: 100
+    )
+    monkeypatch.setattr(
+        "briefcase.platforms.macOS.is_process_dead", lambda *a, **kw: False
+    )
```

Comment:
I struggled a bit to discover a good way to mock these two functions. I ultimately landed here but I'm definitely open to feedback on any best practice you're aware of.